### DISCUSSION TITLE
Unable to install FastAPI on Windows 10 using Python 3.8.1

### DISCUSSION THREAD

**Discussion by codemasta** [fastapi/fastapi#7384] [Questions] [answered]
I create a virtualenv then ran the command 
pip install fastapi[all] but I keep going this error message 

Collecting orjson<4.0.0,>=3.2.1; extra == "all"
  Using cached orjson-3.3.0.tar.gz (654 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... error
 **Caused by:
      process didn't exit successfully: `rustc - --crate-name ___ --print=file-names -Z mutable-noalias --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit code: 1)
    --- stderr**

See screenhot.

![error](https://user-images.githubusercontent.com/1274708/88478424-0eb7bb00-cf40-11ea-9a9e-cc18372a4834.JPG)

**Comment by phy25** [comment, marked as answer]
This is a orjson issue. Might be related to https://github.com/ijl/orjson/issues/33. Please follow the issue template.

**Comment by codemasta** [comment]
> This is a orjson issue. Might be related to [ijl/orjson#33](https://github.com/ijl/orjson/issues/33). Please follow the issue template.

This fixed it. Thanks very much.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
